src/CoreWCF.Primitives/src/CoreWCF/MessageParameterAttribute.cs

```diff
@@ -6,7 +6,7 @@
 namespace CoreWCF
 {
     [AttributeUsage(CoreWCFAttributeTargets.Parameter, Inherited = false)]
-    internal sealed class MessageParameterAttribute : Attribute
+    public sealed class MessageParameterAttribute : Attribute
```

Comment:
this should be public as per https://referencesource.microsoft.com/#System.ServiceModel/System/ServiceModel/MessageParameterAttribute.cs,98a743085573662e ... @mconnew do you have any concern ?

Reply:
No concern, this is the whole point of this pr 😃.